<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_InfoDialogClass</class>
 <widget class="QDialog" name="Admin_InfoDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin_InfoDialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>70</height>
    </size>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 176, 176);</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>771</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="Selected_all">
         <property name="text">
          <string>全选</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pb_add">
           <property name="text">
            <string>添加</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Book/plus-square.svg</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_sub">
           <property name="text">
            <string>取消</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Book/cancel.svg</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete">
           <property name="text">
            <string>删除</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Book/trash-alt.svg</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pb_add_book_file">
         <property name="text">
          <string>批增</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Book/folder-open.svg</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="searchLE"/>
         </item>
         <item>
          <widget class="QCheckBox" name="search_on">
           <property name="text">
            <string>查找</string>
           </property>
           <property name="icon">
            <iconset resource="BookMS.qrc">
             <normaloff>:/Book/search.svg</normaloff>
             <normalon>:/Book/search.svg</normalon>:/Book/search.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>800</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {margin-left: 5px}</string>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::DropAction::IgnoreAction</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>23</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>94</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>27</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>选择</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书籍名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版社</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>馆藏</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出借</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableWidget_stu">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>800</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {margin-left: 5px}</string>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::DropAction::IgnoreAction</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>23</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>94</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>27</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>选择</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>专业</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 176, 176);</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="Selected_all_stu">
         <property name="text">
          <string>全选</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pb_add_stu">
           <property name="text">
            <string>添加</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Book/plus-square.svg</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete_stu">
           <property name="text">
            <string>毕业</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Book/graduation-cap.svg</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pb_add_stu_file">
         <property name="text">
          <string>批增</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Book/folder-open.svg</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>280</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="searchLE_2"/>
         </item>
         <item>
          <widget class="QCheckBox" name="search_on_stu">
           <property name="text">
            <string>查找</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Book/search.svg</normalon>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0px;
background-color: rgb(99, 99, 99);</string>
   </property>
   <widget class="QPushButton" name="pb_page1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color: rgb(99, 99, 99);
font-family:黑体;
font-size:19px;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>图书管理</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Book/book-open-white.svg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_page2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color: rgb(99, 99, 99);
font-family:黑体;
font-size:19px;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>学生管理</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Book/book-reader.svg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BookMS.qrc"/>
  <include location="../../../BookMS by taoyu(With Icon)3/QtWidgetsApplication1/QtWidgetsApplication1/BookMS.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onSearchAction()</slot>
 </slots>
</ui>
